blog_posts/2024-09-12-optimisations-text-more.markdown

```diff
@@ -0,0 +1,142 @@
+---
+title:  "Optimisations, Text Input, Tab Focusing and More!"
+date:   2024-09-12 21:02:00 +0100
+author: Dinnerbone
+icon: /undraw/undraw_setup_analytics_re_foim.svg
+images:
+  - /2024-09-12-optimisations-text-more/club-penguin.png
+---
+We really should start making these posts more often, because phew there's a lot that happened in the last 7 months!
+There's *so many* improvements to Ruffle, that we really can't do it justice with an easy summary here. This blog post is going to be a little long, but the highlights are at the top!
+
+As before, let's summarize with some fancy numbers first.
+
+Since the last blog post (7 month ago)...
+- ActionScript 3 Language has gone up from 75% to **XXXXX%**!
+- ActionScript 3 API has gone up from 68% to **XXXXX%**!
+- ActionScript 1/2 Language has remained at 95%. We're fairly sure we're feature complete, just some pesky bugs left! 
+- ActionScript 1/2 API has gone up from 75% to **XXXXX%**!
+- We've merged **xxxxxxxx** pull requests from **xxxxxxxxxxx** people! (And xxxxxxxxxxx bots.)
+- We've closed **xxxxxxxxx** issues (and some of them were real issues! Crazy!)
+- We've added **xxxxx** new automated tests!
+
+---
+
+### Preparing for a stable release

```

Comment:
Shouldn't these headings be h2 instead of h3?

Reply:
We did h3 in the last posts, I think it looks better but it's not a super strong opinion if you disagree

Reply:
IMO we should also fix these headings in other posts too, h3 not only is semantically weird here, but also looks worse, almost like a part of the text

Reply:
I'll change these to h2 then but the other posts should be its own pr